core/cluster.ts

```diff
@@ -17,3 +19,24 @@ export function getClusterName(cluster: Cluster): { clusterName: string } {
 
   return { clusterName };
 }
+
+export async function getClusterNameFromEndpoint(
+  endpoint: string
+): Promise<{ clusterName: Cluster | 'localnet' | 'unknown' }> {
+  if (endpoint.includes('localhost') || endpoint.includes('127.0.0.1')) {
+    return { clusterName: 'localnet' };
+  }
+
+  const connection = new Connection(endpoint);
```

Comment:
Do you think is worth updating this method [getClusterConnection](https://github.com/lndgalante/solutils/blob/main/core/cluster.ts#L3) or create a new one like `getCustomEndpointConnection` ?

Reply:
Maybe it's worth updating `getClusterConnection` to take string as an argument too but I don't think that's necessary as it would just be a copy of `new Connection`.